lib/shopify_cli/partners_api/app_extensions/thread_pool.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module ShopifyCLI
+  class PartnersAPI
+    class AppExtensions
+      class ThreadPool
```

Comment:
Is this `PartnersAPI`-specific? If not, I recommend placing it under `shopify_cli/thread_pool.rb`

Reply:
Good catch! 🚀 I've extracted `ThreadPool` to `shopify_cli/thread_pool.rb` a new commit :) Thanks for this suggestion!